content/blog/2025-02-10-tmil-13/index.md

```diff
@@ -0,0 +1,97 @@
+
+## Color
+
+[Color][] provides functionality for representing, converting, parsing, serializing, and manipulating colors in a variety of color spaces.
+It closely follows the [CSS Color Module Level 4][] draft spec.
+
+We released [Color 0.2.3][], which added easier methods to convert 8-bit colors from byte streams (for use with GPUs) and to other color representations in [color#135][] and [color#136][].
+This release also saw the addition of the ACES2065-1 color space in [color#124][].
+
+## Vello
+
+Vello is our GPU vector renderer.
+It can draw large 2D scenes with high performance, using GPU compute shaders for most of the work.
+
+We released Vello 0.4.0
+
+- [xilem#791][]: Update to wgpu 0.24
+...
+
+<!-- Screenshot of image extend modes -->
+
+## Parley
+
+Parley is a text layout library.
+It handles text layout, mostly at the level of line breaking and resolving glyph positions.
+In January, a variety of layout edge cases have been fixed, support for bidirectional text was expanded, and testing of text layout and selection has improved.
+
+- [parley#244][]: Adds screenshots to selection area and cursor positioning tests,
+- [parley#241][]: Allows configuring the behavior of alignment for lines that overflow the container width,
+- [parley#245][]: Enables detection of the base direction of text layouts (left-to-right or right-to-left),
+- [parley#249][]: Fixes issues related to line breaking around inline boxes, by new Linebender member Wim de With,
+- [parley#256][]: Corrects the calculation of trailing white space, also by Wim de With,
+- [parley#254][]: Follows the [CSS white space model][] in stripping leading white space following a new line in white space collapsing mode; and
+- [parley#239][]: Takes another step towards supporting `no_std`.
+
+## Kompari
+
+<!-- Repository rework, interactive review, multiple diff methods -->
+
+## Masonry
+
+Masonry is the widget system used by Xilem.
+It provides a non-opinionated retained widget tree, designed as a base layer for high-level GUI frameworks.
+
+<!-- Docs, Box<dyn widget> cleanup, inspector, transforms, zstack -->
+
+## Xilem
+
+Xilem is our flagship GUI project, inspired by SwiftUI.
+It lets you build user interfaces declaratively by composing lightweight views together, and will diff them to provide minimal updates to a retained layer.
+
+<!-- Docs contributed, pod flexibility, emoji picker landed,  -->
+
+## Research and Future Directions
+
+Linebender has an origin story in being a very research oriented group, looking to break new ground.
+While we are focused on shipping code today, we still have an eye on the future and how to be prepared for the new opportunities and technologies that are coming.
+
+We have a couple of active and ongoing research projects:
+
+- Raph Levien is working on a version of Vello which has bounded GPU memory usage. <!-- Codenamed? Potato. Is that the final name? -->
```

Comment:
Here's my proposed text:

While Vello has unmatched rendering speed thanks to its GPU-driven architecture, there are two practical tradeoffs: it can require unpredictable amounts of memory, and it doesn't work on downlevel GPUs with weak or nonexistent support for compute shaders. Raph Levien has been exploring a possible hybrid CPU/GPU direction for Vello to address these issues. Read the [design doc](https://docs.google.com/document/d/1gEqf7ehTzd89Djf_VpkL0B_Fb15e0w5fuv_UzyacAPU/edit?usp=sharing), follow the [Zulip thread](https://xi.zulipchat.com/#narrow/channel/197075-gpu/topic/Potato.20-.20a.20paper.20design.20for.20a.20CPU.2FGPU.20hybrid.20renderer), or stay tuned for more developments.

In response to the comment, I think we're moving away from the "potato" branding.